var/spack/repos/builtin/packages/e4s-cl/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class E4sCl(PythonPackage):
+    """Container Launcher for E4S containers, facilitating MPI library
+    translations"""
+
+    maintainers = ["spoutn1k", "FrederickDeny"]
+    homepage = "https://e4s-cl.readthedocs.io"
+    url = "https://oaciss.uoregon.edu/e4s/e4s-cl/releases"
+    git = "https://github.com/E4S-Project/e4s-cl"
+
+    tags = ["e4s"]
+
+    version("master", branch="master")
+    version("1.0.0", commit="410bb2e6601d9b90243a487ad7f7d2dabd8ba04c")
+    version("1.0.1", commit="b2c92993e0c7cb42de07f0f7cc02da3a06816192")
+
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
Missing a bunch of dependencies:
```
termcolor>=1.1.0
PyYAML>=6.0
texttable>=1.6.2
python-ptrace>=0.9.7
pyelftools==0.27
requests>=2.26.0
tinydb==4.5.2
python-sotools==0.1.0
docker==5.0.3
```

Reply:
Are those dependencies made accessible when installing using `PythonPackage` when listed as spack dependencies ? Just to ensure they are not installed twice.

Reply:
Define "made accessible". Spack uses `pip install` to build and install a wheel, but if the dependencies aren't also installed, the package will be unusable. If you `spack load e4s-cl`, it will also load all dependencies so that they are all in `PYTHONPATH`.

Reply:
Yes and `pip install` also installs those dependencies, so I am asking if listing them via `spack` will make sure they do not get installed twice.

Reply:
Yep, we add the `--no-deps` flag to disable dependency installation so that Spack can manage all deps itself.

Reply:
This should be fixed !